sklearn/semi_supervised/label_propagation.py

```diff
@@ -381,7 +402,10 @@ class LabelSpreading(BaseLabelPropagation):
       parameter for knn kernel
 
     alpha : float
-      clamping factor
+      Clamping factor [0, 1], it specifies the relative amount of the
```

Comment:
A bit more verbose for clarity: "Clamping factor, in [0, 1], specifies the extent to which a sample's label should derive from its neighbors in preference to its initial label."